app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class-level dictionary `people` to store instances of `Person` by name can lead to issues if multiple people have the same name. Consider using unique identifiers or handling name collisions.